packages/renderer/src/lib/container/ContainerColumnAge.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+import StateChange from '../ui/StateChange.svelte';
+import { ContainerUtils } from './container-utils';
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+
+const containerUtils = new ContainerUtils();
```

Comment:
I would argue that we could have static method instead of having a dedicated class for this, I know @benoitf have a preference for it, as it _ease_ the testing _in some cases_ but here I feel like it adds complexity when reading it, as we might think ContainerUtils has internal logic, as it is a class

Reply:
Yeah, I added here b/c well, everything else related is here. Could do static functions in the same file? Easy switch if someone prefers.

Reply:
yeah I would rather have static method where ContainerUtils is defined ?

Reply:
I just review the `containers-utils` file and it should probably be kept like you did for consistency, so forget my comment